docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,392 @@
+
+- The first filter keeps all log records that have the `kubernetes.labels.app` attribute set with value `my-deployment`; all other logs are discarded. The `kubernetes` attribute is available for every log record. See [here](#kubernetes-filter-metadata) for more details.
+- The second filter drops all log records fulfilling the given rule. Here, typical Namespaces are dropped based on the `kubernetes` attribute.
+- A log record is modified by adding a new attribute. Here, a constant attribute is added to every log record for recording the actual cluster node name at the record for later filtering in the backend system. As value, a placeholder is used referring to a Kubernetes-specific environment variable.
+
+### Step 3: Add authentication details from secrets
+
+Integrations into external systems usually need authentication details dealing with sensitive data. To handle that data properly in secrets, the `LogPipeline` supports the reference of Secrets. The key-value entries of the Secrets can be mapped to environment variables of the Fluent Bit Pods, and with that, are available for usage in [placeholder expressions](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables).
+
+To leverage data provided by a Kubernetes Secrets in a custom output definition, use placeholder expressions for the data provided by the Secret and then specify the actual mapping to the keys of the Secret in the `variables` section, like in:
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+  ...
+  output:
+    custom: |
+      Name               http
+      Host               ${ENDPOINT} # Defined in Secret
+      HTTP_User          ${USER} # Defined in Secret
+      HTTP_Password      ${PASSWORD} # Defined in Secret
+      Tls                On
+  variables:
+    - name: ENDPOINT
+      valueFrom:
+        secretKeyRef:
+        - name: http-backend-credentials
+          namespace: default
+          key: ENDPOINT
+    ...
+```
+
+The related secret need to fullfil the referenced name and namespace and need to contain the mapped key as in:
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: http-backend-credentials
+stringData:
+  HTTP_ENDPOINT: https://myhost/logs
+  HTTP_USER: myUser
+  HTTP_PASSWORD: XXX
+```
+
+### Step 4: Rotate the secret
+A Secret being referenced with the `secretKeyRef` construct as used in the previous can be rotated manually or automatically. For automatic rotation, update the actual secret values and keep the keys stable.  
+Once an hour, the LogPipeline watches the referenced secrets and detects changes to them. To enforce the detection, just annotate the LogPipeline; for example, with the following code:
```

Comment:
Is "secret" here unrelated to Kubernetes resources and that's why it's not [capitalized](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms)?